Week2/homework/maartjes-work.js

```diff
@@ -43,19 +43,28 @@ const tuesday = [
 ];
 
 const maartjesTasks = monday.concat(tuesday);
+
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
   // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const mappedTasks = tasks.map(task => task.duration / 60);
+  const filteredTasks = mappedTasks.filter(task => task >= 2);
```

Comment:
This operations should filter the `hours`, such that only hours >= 2 remain. The parameter name `task` is incorrect and misleading here. A correct line would be:

```js
hours = hours.filter(hour => hour >= 2);
```